README.md

```diff
@@ -16,22 +16,21 @@ Documentation for CUBRID RDBMS
 
    1. Install sphinx package and theme
       1. Install python
-      1. Install pip by running command:
 
          ```
-         python get-pip.py
+         yum install python3 -y
          ```
 
-      1. Install sphinx
+      1. Install read the docs theme
```

Comment:
You remove "pip install sphinx" in the readme. 
Can sphinx be installed too with sphinx_rtd_theme?

Reply:
Yes, it can. 
pip3 install sphinx_rtd_theme
......
Installing collected packages: imagesize, MarkupSafe, Jinja2, alabaster, charset-normalizer, idna, certifi, urllib3, requests, pyparsing, packaging, sphinxcontrib-devhelp, pytz, babel, Pygments, sphinxcontrib-applehelp, sphinxcontrib-htmlhelp, docutils, sphinxcontrib-serializinghtml, snowballstemmer, sphinxcontrib-jsmath, sphinxcontrib-qthelp, zipp, typing-extensions, importlib-metadata, **sphinx**, sphinxcontrib-jquery, sphinx-rtd-theme

Successfully installed Jinja2-3.0.3 MarkupSafe-2.0.1 Pygments-2.14.0 alabaster-0.7.13 babel-2.11.0 certifi-2023.7.22 charset-normalizer-2.0.12 docutils-0.18.1 idna-3.4 imagesize-1.4.1 importlib-metadata-4.8.3 packaging-21.3 pyparsing-3.1.1 pytz-2023.3.post1 requests-2.27.1 snowballstemmer-2.2.0 **sphinx-5.3.0** sphinx-rtd-theme-1.3.0 sphinxcontrib-applehelp-1.0.2 sphinxcontrib-devhelp-1.0.2 sphinxcontrib-htmlhelp-2.0.0 sphinxcontrib-jquery-4.1 sphinxcontrib-jsmath-1.0.1 sphinxcontrib-qthelp-1.0.3 sphinxcontrib-serializinghtml-1.1.5 typing-extensions-4.1.1 urllib3-1.26.16 zipp-3.6.0

Reply:
pip3 install sphinx
Requirement already satisfied: sphinx in /usr/local/lib/python3.6/site-packages
.....

Reply:
OK, i see